client-src/clients/SockJSClient.js

```diff
@@ -11,7 +11,9 @@ module.exports = class SockJSClient extends BaseClient {
     super();
     this.sock = new SockJS(url);
 
-    this.sock.onerror = () => {};
+    this.sock.onerror = (err) => {
+      console.error(err);
```

Comment:
We should use logger, instead `console`

Reply:
@evilebottnawi This is another case where the path from `SockJSClient` to `utils/log` is not the same in the `client-src` directory compared to the `client` directory. I think we should wait and add this after reorganizing `client` directories on `next`, rather than do more of this redirection: https://github.com/webpack/webpack-dev-server/blob/c2da5322b078dbe9c3f6580ca2f301ba3e282452/client-src/default/webpack.config.js#L21.

Reply:
Let me remove the `console.error` for now, since it did not exist before and we won't introduce anything new in console output.